src/main/java/appeng/items/tools/ToolNetworkTool.java

```diff
@@ -219,13 +219,11 @@ public boolean isUsable( ItemStack item, EntityLivingBase user, Entity entity )
 	@Override
 	public void toolUsed( ItemStack item, EntityLivingBase user, BlockPos pos )
 	{
-		user.swingArm( user.getActiveHand() );
 	}
 
 	@Override
 	public void toolUsed( ItemStack item, EntityLivingBase user, Entity entity )
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1186)